pytorch_lightning/accelerators/accelerator.py

```diff
@@ -52,21 +52,16 @@ class Accelerator:
 
     def __init__(
         self,
-        precision_plugin: PrecisionPlugin,
         training_type_plugin: TrainingTypePlugin,
     ) -> None:
         """
         Args:
             precision_plugin: the plugin to handle precision-specific parts
             training_type_plugin: the plugin to handle different training routines
         """
-        self.precision_plugin = precision_plugin
+        self.precision_plugin = training_type_plugin.precision_plugin
```

Comment:
do we even need a fixed reference here then? I think replacing it with a property might be better since that does not increase the reference counter of the object